recipes-nodes/rbuilder-toggle/rbuilder-toggle.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/sh
+### BEGIN INIT INFO
+# Provides:          rbuilder-toggle
+# Required-Start:    $network
+# Required-Stop:
+# Default-Start:     2 3 4 5
+# Default-Stop:      0 1 6
+# Short-Description: rbuilder status watchdog
+# Description:       Checks the status of rbuilder in BuilderHub and toggles the service
+### END INIT INFO
+
+set -e
+
+SYSTEM_API_FIFO="/var/volatile/system-api.fifo"
+RBUILDER_PERSISTENT_DIR="/persistent/rbuilder"
+RBUILDER_STATUS_FLAG_FILE="${RBUILDER_PERSISTENT_DIR}/rbuilder.enabled"
+BHUB_URL="http://localhost:7937/api/l1-builder/v1/configuration"
+
+log() {
+    if [ -p $SYSTEM_API_FIFO ]; then
+        date_log() {
+            echo -n ""
+        }
+    else
+        date_log() {
+            echo -n "$(date --iso-8601=seconds): "
+        }
+    fi
+    echo "$(date_log)$1" | tee -a $SYSTEM_API_FIFO
+}
+
+case "$1" in
+  start)
+    BHUB_CONFIG=$(curl -fsSL --retry 3 --retry-delay 5 --retry-connrefused $BHUB_URL)
+
+    if [ -z "${BHUB_CONFIG}" ]; then
+      log "rbuilder-toggle: Failed to fetch configuration."
+      exit 1
+    fi
+
+    TOGGLE_STATUS=$(echo "${BHUB_CONFIG}" | jq -r '.rbuilder.enabled')
```

Comment:
```suggestion
    RBUILDER_SHOULD_BE_ENABLED=$(echo "${BHUB_CONFIG}" | jq -r '.rbuilder.enabled')
```

more descriptive variable name would be nice

Reply:
renamed to `RBUILDER_ENABLED` to be more descriptive